3MI0800270/2023-03-07/zad5.cpp

```diff
@@ -0,0 +1,335 @@
+#include <iostream>
+#include <fstream>
+#include <cstring>
+using namespace std;
+
+const char FILE_PATH[10] = "zad5.txt";
+
+struct User {
+    char username[33];
+    char password[33];
+    char email[33];
+};
+
+int getTableSize(const char* filepath) {
+    ifstream file(filepath);
+    int count = 0;
+    while (!file.eof()) {
+        char ch = file.get();
```

Comment:
Reading symbols one by one is everything but a good idea. You better refer to using [getline](https://cplusplus.com/reference/istream/istream/getline/) instead